end-to-end-test/local/specs/core/groupComparison.screenshot.spec.js

```diff
@@ -158,10 +159,13 @@ describe('group comparison page screenshot tests', function() {
         });
 
         it('group comparison alteration enrichments two groups', function() {
-            goToUrlAndSetLocalStorage(
-                `${browser.getUrl()}&unselectedGroups=%5B"GB"%2C"OAST"%2C"ODG"%5D`,
-                true
-            );
+            // this test will not work on retry because groups will be toggled back on
+            this.retries(0);
+
+            getElementByTestHandle('groupSelectorButtonGB').click();
```

Comment:
i did this to make it a little more realistic than just editing url to shut off the groups.